alf/algorithms/hypernetwork_algorithm.py

```diff
@@ -122,6 +129,24 @@ def __init__(self,
 
             critic_optimizer (torch.optim.Optimizer): the optimizer for training critic.
             critic_hidden_layers (tuple): sizes of critic hidden layeres.
+            critic_iter_num (int): number of minmax optimization iterations to 
+                train critic
+            critic_l2_weight (float): L2 penalty on critic to ensure
+                boundednesss
```

Comment:
It seems that critic_l2_weight is not used?

Reply:
critic_l2_weight is passed to the generator, that's its only function in hypernetwork_algorithm